cmd/cmd.go

```diff
@@ -1,93 +1,104 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
-	"strings"
 	"time"
 
 	"github.com/Clever/batchcli/runner"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/sfn"
 )
 
 func main() {
-	functionCmd := flag.String("cmd", "", "The command to run")
-	localRun := flag.Bool("local", false, "Local mode - auto-assigns random AWS Batch config")
+	activityName := flag.String("name", "", "The activity name to register with AWS Step Functions")
+	activityCmd := flag.String("cmd", "", "The command to run to process activity tasks")
+	sfnRegion := flag.String("region", "", "The AWS region to send Step Function API calls")
 	printVersion := flag.Bool("version", false, "Print the version and exit")
-	resultsLocation := flag.String("results-location", "workflow-results-dev", "name of the table for getting and setting job results")
```

Comment:
I think it's useful to store the results using an `JobID, AppName, Success/Failure, Output` for any system we end up using.

Reply:
in a world where step functions stores all of this and also takes care of passing outputs as inputs to further states, what's the point of mirroring all of this in dynamo?

Reply:
I could see mirroring some of the data for an execution history in `workflow-manager`, but doing it here in the cli seems like overkill

Reply:
AFAIK there is no straightforward way to look up outputs of individual activities. If that is possible then no need. 

The reason I was proposing saving it in dynamo was for workflow manager to be able to get outputs from individual activities.

Reply:
the execution history API has the [TaskStateEntered](https://docs.aws.amazon.com/step-functions/latest/apireference/API_StateEnteredEventDetails.html) event type and [TaskStateExited](https://docs.aws.amazon.com/step-functions/latest/apireference/API_StateExitedEventDetails.html) event types that show the input / output of states: https://gist.github.com/rgarcia/b64bafdc43601aca3bfa20b5074ed37f

Reply:
wahoo!! Awesome. Excited then that we don't have to have another datastore involved :)